.github/workflows/nightly-rockset-uploads.yml

```diff
@@ -33,4 +33,28 @@ jobs:
           retry_wait_seconds: 90
           command: |
             echo "Uploading external contribution stats for" "$(date -d yesterday '+%Y-%m-%d')"
-            python3 -m tools.stats.upload_external_contrib_stats --startDate "$(date -d yesterday '+%Y-%m-%d')"
\ No newline at end of file
+            python3 -m tools.stats.upload_external_contrib_stats --startDate "$(date -d yesterday '+%Y-%m-%d')"
+
+  upload-testing-stats-aggregates:
+    runs-on: [self-hosted, linux.2xlarge]
+    steps:
+      - name: Checkout PyTorch
+        uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
```

Comment:
Just FYI, you could try checkout with `fetch-depth: 1` here I think to make this step faster.  PyTorch checkout action default to pull all repo history.  Checking out PyTorch probably takes more time to run then the command itself :)

Reply:
It indeed does haha

Reply:
It looks a bit wrong.  There should be with params part, i.e. https://github.com/pytorch/pytorch/blob/73b06a0268bb89c09a86f16fa0f72818baa4b250/.github/workflows/check-labels.yml#L18-L21

```
uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
with:
  submodules: false
  fetch-depth: 1
```

Does lint bark on this?

Reply:
It does :)